### ISSUE TITLE
episode 02-practice-learning - analogy

### ISSUE THREAD

**Issue by yabellini** [carpentries/instructor-training#1323]
The episode says:  consider this common analogy from education: "teaching is like gardening."

This is not a common analogy in Latin America. I did a little research and asked several people, and on social media, most people never heard this phrase, and the interpretation is pretty different. From positive aspects to not-so-positive feelings about comparing gardening and teaching.

So perhaps we can remove the word _common_ from the text.

**Comment by anenadic** [comment]
I agree with this - I was going to suggest to expand on and explain a bit more the analogy (which is a great example) for instructors who are not perhaps as familiar with it and also to suggest we include a few other ones provided by the community under the solution of the exercise but you beat me too it @yabellini. I might start a separate issue about the solution.

**Comment by Rasundsbak** [comment]
If teaching is like gardening, we may say that you as a teacher, will need to be patient, and just place the seeds of knowledge in your students. You may do this without seeing any immediate results. The learning material needs to mature, and the blossoming might take place long after the course, and at a different time and place than in the classroom...

**Comment by jonahcullen** [comment]
I found it a little challenging to come up with a good analogy on the spot during the instructor training workshop. I was wondering if it might be beneficial to include this as a pre-workshop suggested task?

**Comment by yabellini** [comment]
It seems that analogies have a component of regional, cultural and prior knowledge about a topic.  It is effective if the person knows about the topic one is using the analogy about because the objective is to use it to explain the new concept.  I don't know how effective it is if I have to explain the analogy (although it can be interesting if we debate the validity of the analogy).

I think it is a good idea to add other examples to have options to select the analogy that works best for our audience.

**Comment by brownsarahm** [comment]
@jonahcullen where would you expect a task like that to be? 

💯 @yabellini that analogies are tough with audiences without a universally shared culture to draw from.  I think a little bit of expansion on the analogy in the curriculum could help at least the trainer know what the goal of the analogy is so that they can find something more audience appropriate if possible? For member-site workshops, the host could provide information that would help the trainer. For online trainings with open seats, it may not be possible to find one that works for every single person, but the expanded text would at least help make the point more complete.  

Removing common is definitely a good choice. A short term PR for that alone would be welcome.

**Comment by SoniaBarasa** [comment]
I agree that it was a bit of a challenge to come up with an analogy example instantly and got most learners scratching their heads and looking stuff up. If the participants would be informed prior as part of what to prepare on so they familiarize themselves ,it would not only save the practice time but also so they have a clear understanding and context in which to use an analogy appropriately.
